<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReqPiece</class>
 <widget class="QDialog" name="ReqPiece">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Réquisition de Pièce</string>
  </property>
  <widget class="QPushButton" name="valider_pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>830</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler_pushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>830</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>179</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="equipement_titles_verticalLayout">
    <item>
     <widget class="QLabel" name="ID_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cat_equip_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Catégorie d'équipement:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="marque_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Marque:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="modele_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Modèle:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unite_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Unité:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="salle_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Salle:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nom_tech_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nom du technicien:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="date_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>251</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="equipement_values_verticalLayout">
    <item>
     <widget class="QLabel" name="ID_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cat_equip_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="marque_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="modele_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unite_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="salle_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nom_tech_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="date_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="function_title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Réquisition de pièce</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enregistrer_pdf_checkBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>770</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enregistrer en PDF</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="envoyer_courriel_checkbox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>770</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Envoyer par courriel</string>
   </property>
  </widget>
  <widget class="QLabel" name="equipement_title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Équipement</string>
   </property>
  </widget>
  <widget class="QLabel" name="piece_associee_title_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Pièce associée</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>160</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="piece_titles_verticalLayout">
    <item>
     <widget class="QLabel" name="cat_piece_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Catégorie de la pièce:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nom_piece_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nom de la pièce:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nbr_unites_title_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre d'unités:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>120</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="piece_values_verticalLayout">
    <item>
     <widget class="QLabel" name="cat_piece_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nom_piece_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nbr_unites_label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="desc_situation_title_label">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>300</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description de la situation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_intervention_title_label">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>460</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description de l'intervention:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="desc_situation_verticalLayout">
    <item alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QLabel" name="desc_situation_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="desc_intervention_verticalLayout">
    <item alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QLabel" name="desc_intervention_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="photo_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>421</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="photo_piece_title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Photo de la pièce:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
